README.md

```diff
@@ -68,22 +68,36 @@ $ export KUBECONFIG=/path/to/admin.kubeconfig
   ```
   $ make run-local
   ```
-4. Open a new terminal window and create an Infinispan cluster with three nodes:
+4. Optionally install the Infinispan operator on OKD from the bundle file.
+  * Create a new project or use an existing one:
+  ```
+  $ oc new-project infinispan-operator
+  ```
+  * If your project name different from the `infinispan-operator` (in this example `my-infinispan-operator`), download operator install bundle to the local store and replace namespace with the following command:
```

Comment:
```suggestion
  * If your project name differs from `infinispan-operator` (in this example `my-infinispan-operator`), you need to download the operator install bundle file to the local store and replace the predefined namespace with the following command:
```

Reply:
If you're supposing that the user doesn't checkout the whole project, you probably need to provide info on how to get the `cr_minimal.yaml` below in point 5.

Reply:
maybe @oraNod can provide a better rephrase for the param above.

Reply:
> If you're supposing that the user doesn't checkout the whole project, you probably need to provide info on how to get the `cr_minimal.yaml` below in point 5.

@rigazilla let's point to the raw.githubusercontent.com whether local or remote creating

Reply:
ok